mesosdriver/executor_driver.go

```diff
@@ -0,0 +1,318 @@
+package mesosdriver
```

Comment:
This is 100% new code, and is the new custom "driver" that makes callbacks into `LaunchTask()` and `KillTask()` in the `sidecarExecutor`.

Reply:
I'm a bit concerned about the `context.TODO()` calls in there. I need to check the code in detail, but I wonder if we could end up blocking forever in some situations.

Reply:
This is all a bit shit, not sure why they expect contexts to be passed in for some of these calls. They all rely on the underlying HTTP driver layer, which itself has a timeout set. Allowing two different layers of timeouts is a recipe for a bad life, so I've left these as TODO calls.